### DISCUSSION TITLE
Is it possible to configure @floating-ui middlewares deeply?

### DISCUSSION THREAD

**Discussion by Marsony** [mantinedev/mantine#5834] [Q&A]
Hello! Thank you very much for the wonderful library.

Please tell me, is it now possible to configure middlewares for Popover/Tooltip more deeply? According to the documentation, we can currently just enable or disable flip/shift/inline. But I would like to configure these aspects more deeply, for example, to manage the boundaries of components on mobile devices. Thank you very much!

Example:
```
  const data = useFloating({
    placement,
    open,
    onOpenChange: setOpen,
    whileElementsMounted: autoUpdate,
    middleware: [
      offset(5),
      flip({
        crossAxis: placement.includes("-"),
        fallbackAxisSideDirection: "end",
        padding: 5
      }),
      shift({ padding: 5 })
    ]
  });
```

Here is we can control paddings of the flip/shift/etc

**Comment by rtivital** [comment]
No, currently, it is not supported
